src/components/page/Header/index.tsx

```diff
@@ -216,6 +219,19 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
                         </DropdownItem>
                       </div>
                     )}
+                    {ENABLE_IRS_PERFORMANCE_REPORT && (
```

Comment:
if this is the only thing enabled in `monitors` nav tab, should the tab be rendered? i.e. condition at l194